### PR TITLE
fix ticket:4209, suppress wrong error messages

### PR DESCRIPTION
**Pull request by adrpo**

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2017-01-09_23-27-46](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3639/console).

File: Compiler/Script/Interactive.mo [hunk 1]
```diff
@@ -15620,7 +15644,7 @@ algorithm
   end match;
 end deleteProtectedList;
 
-protected function getPublicList "
+public function getPublicList "
```
Comment (lochel) [inline Compiler/Script/Interactive.mo]: Why did you make this and function `getProtectedList` public?

Comment (adrpo) [inline Compiler/Script/Interactive.mo]: I need to use that in another file for some other commit I'm working on.

**Comment by casella** [comment]
@adrpo would you mind merging this into 1.11 as well?

**Comment by adrpo** [comment]
@casella: is done!

**Comment by casella** [comment]
Thanks!

**Pull request closed without merging**